File: Modul 4/Soal 5/PRAK405-2410817210019-AchmadReihanAlfaiz.py
n, multiplier = map(int, input().split())
grand_total = 0

for i in range(1, n + 1):
    sum_value = (i * (i + 1) // 2) * multiplier
    grand_total += sum_value

    for j in range(1, i + 1):
        if j > 1:
            print(" + ", end="")
        print(f"({i - j + 1} * {multiplier})", end="")
    print(f" = {sum_value}")

print(grand_total)

File: Modul 4/Soal 5/PRAK405-2410817210019-AchmadReihanAlfaiz.c
#include <stdio.h>

int main()
{
     int n, multiplier, grandTotal = 0;
     scanf("%d %d", &n, &multiplier);

     for (int i = 1; i <= n; i++)
     {
          int sum = (i * (i + 1) / 2) * multiplier;
          grandTotal += sum;

          for (int j = 1; j <= i; j++)
               printf("%s(%d * %d)", j > 1 ? " + " : "", j > 1 ? i - j + 1 : i, multiplier);
          printf(" = %d\n", sum);
     }

     printf("%d\n", grandTotal);
     return 0;
}